src/CMakeLists.txt

```diff
@@ -768,6 +768,10 @@ rocm_set_soversion(MIOpen ${MIOpen_SOVERSION})
 
 clang_tidy_check(MIOpen)
 
+if(HAS_LIB_STD_FILESYSTEM)
+    target_link_libraries(MIOpen PRIVATE stdc++fs)
```

Comment:
It seems to be only necessary at some legacy Linux distros and GCC 8?

Reply:
Yes, that's right. We could put conditionals on the compiler being `GCC` and version being < 8.

Reply:
actually, we can't do that because our compiler is `clang++`. Rather this conditional has to go on something else, e.g., a version of `libstdc++`, which I don't know how to detect, so `HAS_LIB_STD_FILESYSTEM` is a good enough choice till then because it should only exist on older systems (I hope).

Reply:
Here's some extra details:
- https://bugs.launchpad.net/ubuntu/+source/gcc-8/+bug/1824721

I think this will be fine on newer systems, and we are utilizing the solution from the original PR:
- https://github.com/ROCm/MIOpen/pull/2690